### PR TITLE
Remove unnecessary eval

### PR DESCRIPTION
**Pull request by Krakn**
Removes an unnecessary call to eval from the travis-setup.sh file.

EDIT: Removed another eval from the common.sh file.

### CODE REVIEW

**Comment by Krakn** [comment]
I'm not entirely sure what I changed that broke the translation report...

File: ci/common/common.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # ${3}: Line number.
 require_environment_variable() {
   local variable_name="${1}"
-  eval "local variable_content=\"\${${variable_name}:-}\""
+  local variable_content="${variable_name:-}"
```
Comment (justinmk) [inline ci/common/common.sh]: @ZyX-I Added this to workaround `set -u`.

Comment (ZyX-I) [inline ci/common/common.sh]: The code is entirely different. Before `$variable_content` was set to variable content, not surprisingly. Now you set it to variable name.

Comment (ZyX-I) [inline ci/common/common.sh]: So basically this function is now as good as

```Shell
require_environment_variable() {
  true
}
```

File: scripts/travis-setup.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ _setup_deps() {
   export NVIM_DEPS_PREFIX="${1}/usr"
   echo "\$NVIM_DEPS_PREFIX: \"${NVIM_DEPS_PREFIX}\""
 
-  eval "$(${NVIM_DEPS_PREFIX}/bin/luarocks path)"
+  "$(${NVIM_DEPS_PREFIX}/bin/luarocks path)"
```
Comment (ZyX-I) [inline scripts/travis-setup.sh]: This one is also different. Before it was setting LUA_PATH and LUA_CPATH. Now you are trying to execute `export LUA_PATH=…\nexport LUA_CPATH=…` *as a single command* and, of course, this is not going to work.

**Comment by ZyX-I** [comment]
Do not know what you tried to achieve, but this PR only breaks things and should be closed.

File: ci/common/common.sh (hunk 1 above)
Comment (Krakn) [inline ci/common/common.sh]: @ZyX-I 

Thanks for the explanation!

**Pull request closed without merging**